Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/database/CommonConverters.kt

```diff
@@ -78,4 +79,10 @@ class CommonConverters {
 
     @TypeConverter
     fun fromLocationCode(code: LocationCode?): String? = code?.identifier
+
```

Comment:
Are we not using `Instant` for the database, why do we need Date` in an unrelated converter?

Reply:
true. was relict from older commits.
cleaned.